samples/.eslintrc.yml

```diff
@@ -1,3 +1,5 @@
 ---
 rules:
   no-console: off
+  node/no-unpublished-require: off
```

Comment:
Note, this file is auto-generated. Then next time `npm run generate-scaffolding` is run, these changes would be overwritten.

Your options are:

1) Add `--except eslintrc_samples` to the `generate-scaffolding` script, or
2) Talk to @lukesneeringer about getting your changes incorporated into the [eslintrc_samples template in nodejs-repo-tools](https://github.com/GoogleCloudPlatform/nodejs-repo-tools/blob/master/templates/nodejs/eslintrc_samples.tpl)

Reply:
removed.